### PR TITLE
chore(index.html): fixed grammer in comment

### PR DESCRIPTION
**Pull request by henryw4k**
Made comment more clear by cleaning up the grammar.

### CODE REVIEW

File: test/index.html [hunk 1]
```diff
@@ -40,7 +40,7 @@
 	<div id="qunit"></div>
 
 	<!-- Test HTML -->
-	<!-- this iframe is outside the #qunit-fixture so it won't reload constantly wasting time, but it means the tests must be "safe" and clean up after themselves -->
+	<!-- this iframe is outside the #qunit-fixture so it won't waste time by constantly reloading; the tests are "safe" and cleans up after themselves -->
```
Comment (gibson042) [inline test/index.html]: "cleans" → "clean"

**Comment by gibson042** [comment]
We might want to fix the spelling error in the commit subject when landing, unless an intentional nod to [Hartman's Law of Prescriptivist Retaliation](https://en.wikipedia.org/wiki/Muphry%27s_law) is desired.

**Comment by markelog** [comment]
@gibson042 loved the link :-)

**Comment by henryw4k** [comment]
:)

**Comment by mzgol** [comment]
@henryw4k Why did you close this PR?

**Comment by henryw4k** [comment]
@mzgol : doesn't look like anyone was going to merge it

**Comment by markelog** [comment]
We will!

**Comment by henryw4k** [comment]
ohhh i see. thanks!

**Pull request closed without merging**